cylc/flow/scripts/cylc_help.py

```diff
@@ -345,15 +344,14 @@ def category_help(category):
 comsum['run'] = 'Start a suite at a given cycle point'
 comsum['stop'] = 'Shut down running suites'
 comsum['restart'] = 'Restart a suite from a previous state'
-comsum['trigger'] = 'Manually trigger or re-trigger a task'
-comsum['insert'] = 'Insert tasks into a running suite'
-comsum['remove'] = 'Remove tasks from a running suite'
+comsum['spawn'] = 'Spawn tasks that depend on outputs'
```

Comment:
There is a comsum['spawn'] also on line 354 (with a slightly different description), are both of these needed?